crates/cli/src/cli.rs

```diff
@@ -186,6 +185,11 @@ pub struct Cli {
     /// Option for system contracts (default: built-in).
     pub dev_system_contracts: Option<SystemContractsOptions>,
 
+    #[arg(long, value_parser = protocol_version_from_str, help_heading = "System Configuration")]
+    /// Protocol version to use for new blocks (default: 26). Also affects revision of built-in
```

Comment:
You won't need to hardcode default if you use `default_value_t` instead of `Option`. Then the help message would contain something like this ([playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2024&gist=0e41e7de86153bad24f7934205e6ed88)):

```
Options:
      --val <VAL>                [default: 1]
```

Reply:
That was my first instinct as well. But, with the way telemetry is implemented right now, having a non-optional field with default will make telemetry report as if the user provided this themselves (see https://github.com/matter-labs/anvil-zksync/blob/main/crates/cli/src/cli.rs#L729). So I just went with the established design for now... Not sure how to even refactor this without overhauling the whole thing.